default-config.json

```diff
@@ -310,62 +310,9 @@
           "type": "bool", 
           "description": "Loop",
           "default": true
-        },
-        "audioType": {
```

Comment:
Slightly concerned people will lose their settings for these when upgrading to this version of the addon.. but not sure its worth writing special code to migrate this.. hmm